### ISSUE TITLE
Merge Request Query for negate

### ISSUE THREAD

**Issue by mochadwi** [TeamCodeStream/codestream#677]
<img width="805" alt="Screen Shot 2021-08-04 at 5 45 53 PM" src="https://user-images.githubusercontent.com/7057959/128168301-a1e8c5be-cdc4-40bd-af28-e55dde47ea1c.png">

is it possible to query the `not` in CodeStream? Also, the `not` was receiving `Hash` instead of `string`, there's no official docs/example how to query `not` through API as well

**Comment by mochadwi** [comment]
```
Request URL: https://gitlab.com/<group>/<repo>/-/merge_requests?scope=all&state=opened&not[label_name][]=prd054&not[label_name][]=prd052
```

Looks like using negation `not` is like the above request, is it possible to have this query syntax in CodeStream?

**Comment by planteater** [comment]
@mochadwi - Good timing! In our next release we'll be updating the MR query syntax to match that on GitLab, and with it will come support for `not`. I'll update you here when the release is out. Probably sometime next week.

**Comment by planteater** [comment]
This should be addressed in v11.0.13 of CodeStream's extension for JetBrains.
